web/src/components/organisms/Asset/AssetList/index.tsx

```diff
@@ -1,9 +1,12 @@
 import styled from "@emotion/styled";
 import { useParams } from "react-router-dom";
 
+import PageHeader from "@reearth-cms/components/atoms/PageHeader";
```

Comment:
Can you make an AssetList molecule so you can move the styled Wrapper out of the organism? If you do that, you can do `<AssetListMolecule {all your props here} />` in this file and then in the molecule you can import and use the PageHeader, AssetListTable, etc.

Reply:
Done!